packages/typescript-estree/src/parseSettings/getProjectConfigFiles.ts

```diff
@@ -0,0 +1,63 @@
+import debug from 'debug';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import type { ParseSettings } from '.';
+
+const log = debug('typescript-eslint:typescript-estree:getProjectConfigFiles');
+
+const tsconfigMatchCache = new Map<string, string | undefined>();
```

Comment:
nit global const variable naming

```suggestion
const TSCONFIG_MATCH_CACHE = new Map<string, string | undefined>();
```